bin/build

```diff
@@ -25,7 +25,12 @@ version() {
 
 frontend-deps() {
     echo "Running 'yarn' to download javascript dependencies..." &&
-    yarn
+    if [[ -z "${CI}" ]]; then
```

Comment:
Is `"${CI}"` different from `$CI`?

Reply:
> Is `"${CI}"` different from `$CI`?

In this case (which I wrote out of habit), there is no difference since there is no variable expansion.